scripts/zones/Nyzul_Isle/instances/nashmeiras_plea.lua

```diff
@@ -35,15 +35,15 @@ function onInstanceProgressUpdate(instance, progress)
     if (progress == 4) then
         local chars = instance:getChars()
 
-        for i, v in pairs(chars) do
-            v:startEvent(203)
-        end
-
         DespawnMob(ID.mob[59].RAZFAHD, instance)
         SpawnMob(ID.mob[59].ALEXANDER, instance)
 
     elseif(progress == 5) then
         instance:complete()
+		v:setPos(126.6543, 0, 20.1798, 0, 72)
+		for i, v in pairs(chars) do
+			v:startEvent(203)
+        end
```

Comment:
Is this cs supposed to play here? [This video](https://youtu.be/SLr-slWFjxQ?t=954) looks like its supposed to play when Alexander spawns.